app/login/register.controller.spec.js

```diff
@@ -0,0 +1,86 @@
+describe('loginController', function () {
```

Comment:
rename `register-controller.js` to `login-controller.js`, check all dependencies

Reply:
done